shared/src/packages/business/libs/types/request/business-entity.type.ts

```diff
@@ -0,0 +1,8 @@
+type BusinessEntity = {
+  id: number;
+  companyName: string;
+  taxNumber: string;
+  ownerId: string;
```

Comment:
ownerId: number ?

Reply:
sure number, and not ownerId but userId, fixed